deploy/example/mountstorage/README.md

```diff
@@ -0,0 +1,276 @@
+    ```bash
+    aksnprg="$(az aks list -g "$resourcegroup" --query "[?name == '$aksname'].nodeResourceGroup" -o tsv)"
+    aksnp="$(az vmss list -g "$aksnprg" --query "[?starts_with(name, 'aks-nodepool1-')].name" -o tsv)"
+    miid="$(az identity list -g "$resourcegroup" --query "[?name == 'myaksblobmi'].id" -o tsv)"
+    az vmss identity assign -g "$aksnprg" -n "$aksnp" --identities "$miid"
+    ```
+
+1. Get the clientID of your user-assigned managed identity
+    ```bash
+    az identity list -g "$resourcegroup" --query "[?name == 'myaksblobmi'].clientId" -o tsv
+    ```
+
+## Mount the azure blob storage
+
+1. Create a ``volume.yaml`` file and set clientID for ``AzureStorageIdentityClientID``. \
+   Please also check ``resourceGroup`` and ``storageAccount``.
+    ```yml
+    apiVersion: v1
+    kind: PersistentVolume
+    metadata:
+      name: pv-blob1
+    spec:
+      capacity:
+        storage: 10Gi
+      accessModes:
+        - ReadWriteMany
+      persistentVolumeReclaimPolicy: Retain  # If set as "Delete" container would be removed after pvc deletion
+      storageClassName: azureblob-fuse-premium
+      mountOptions:
+        - -o allow_other
+        - --file-cache-timeout-in-seconds=120
+      csi:
+        driver: blob.csi.azure.com
+        readOnly: false
+        # make sure this volumeid is unique in the cluster
+        # `#` is not allowed in self defined volumeHandle
+        volumeHandle: pv-blob1
+        volumeAttributes:
+          protocol: fuse
+          resourceGroup: aks-fuseblob-mi
+          storageAccount: myaksblob
+          containerName: mycontainer
+          AzureStorageAuthType: MSI
+          AzureStorageIdentityClientID: "xxxxx-xxxx-xxx-xxx-xxxxxxx"
+    
+    ---
+    
+    apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      name: pvc-blob1
+    spec:
+      accessModes:
+        - ReadWriteMany
+      resources:
+        requests:
+          storage: 10Gi
+      volumeName: pv-blob1
+      storageClassName: azureblob-fuse-premium
+    ```
+
+1. Create a ``deployment.yaml`` file.
+    ```yml
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      labels:
+        app: nginx-app1
+      name: nginx-app1
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: nginx-app1
+      template:
+        metadata:
+          labels:
+            app: nginx-app1
+        spec:
+          containers:
+          - image: mcr.microsoft.com/oss/nginx/nginx:1.19.5
+            name: webapp
+            imagePullPolicy: Always
+            resources: {}
+            ports:
+              - containerPort: 80
+            volumeMounts:
+              - name: pvc-blob1
+                mountPath: /usr/share/nginx/html
+          volumes: 
+            - name: pvc-blob1 
+              persistentVolumeClaim: 
+                claimName:  pvc-blob1
+    status: {}
+    
+    ---
+    
+    apiVersion: v1
+    kind: Service
```

Comment:
let's remove this example, make it simple

Reply:
done